### PR TITLE
Allow returning "Unknown" orientation from Android

### PR DESCRIPTION
**Pull request by nwparker**
# What is this change
This PR handles the [ORIENTATION_UNKNOWN](https://developer.android.com/reference/android/view/OrientationEventListener#ORIENTATION_UNKNOWN) state returned by Android and returns the "Unknown" orientation instead of the previous default to "PortraitUp"

To preserve previous functionality, each orientation getter/listener method now has an included `unknownSubstitute` parameter. The default value is `portraitUp` which should preserve the current behavior, but allows developers to change this at their will.

# Why this way
I opted to do the logic of substitution in Dart as opposed to Java as I feel additional logic should live there. The Java code with this change returns as superset of information (as opposed to before this change). Writing native code is generally more of a escape hatch when necessary for functionality or when performance deems the situation as required. Neither of these cases apply here.

# Background
See discussion https://github.com/rmtmckenzie/flutter_native_device_orientation/pull/12#issuecomment-734114697
tl;dr : Some developers need to know when Android phones are flat

### CODE REVIEW

**Comment by nwparker** [comment]
bumping for review

File: lib/native_device_orientation.dart [hunk 1]
```diff
@@ -53,8 +58,11 @@ class NativeDeviceOrientationCommunicator {
     await _methodChannel.invokeMethod('resume');
   }
 
-  Stream<NativeDeviceOrientation> onOrientationChanged({bool useSensor = false}) {
-    if (_stream == null || _stream!.useSensor != useSensor) {
+  Stream<NativeDeviceOrientation> onOrientationChanged({
+    bool useSensor = false,
+    NativeDeviceOrientation unknownSubstitute = NativeDeviceOrientation.portraitUp,
+  }) {
+    if (_stream == null || _stream.useSensor != useSensor) {
```
Comment (nwparker) [inline lib/native_device_orientation.dart:65]:
```suggestion
    if (_stream == null || _stream!.useSensor != useSensor) {
```

**Comment by rmtmckenzie** [comment]
Sorry for the longest wait ever on this, but it's finally merged into v1.1.0. I did change the name to defaultOrientation though.
